src/tronls.jl

```diff
@@ -99,12 +159,16 @@ function tron(
   start_time = time()
   el_time = 0.0
 
+  solver.x .= x
+  x = solver.x
+  gx = solver.gx
+
   # Preallocation
-  Av = zeros(T, m)
-  Atv = zeros(T, n)
-  gpx = zeros(T, n)
-  xc = zeros(T, n)
-  Hs = zeros(T, n)
+  Av = solver.Av
+  Atv = solver.Atv
+  gpx = solver.gpx
+  xc = solver.xc
+  Hs = solver.Hs
```

Comment:
Do you use `solver.H` and `solver.Hs` in the code?

Reply:
You are right this wasn't used! Thanks.